mobile/android/android-device.ts

```diff
@@ -259,6 +259,9 @@ export class AndroidDevice implements Mobile.IAndroidDevice {
         var clearDebugEnvironmentCommand = this.composeCommand('shell rm "%s"', envDebugOutFullpath);
         this.$childProcess.exec(clearDebugEnvironmentCommand).wait();
 
+		var setDebugBreakEnvironmentCommand = this.composeCommand('shell echo "" > "%s"', "debugbreak");
```

Comment:
Can you use ``` touch %s``` instead?

Also, it looks like there are mixed tabs / spaces, please fix them.